press/press/doctype/database_server/database_server.py

```diff
@@ -603,7 +603,7 @@ def adjust_memory_config(self):
 		self.add_mariadb_variable(
 			"innodb_buffer_pool_size",
 			"value_int",
-			int(self.ram_for_mariadb * 0.685),  # will be rounded up based on chunk_size
+			int(self.ram_for_mariadb * 0.65),  # will be rounded up based on chunk_size
```

Comment:
We can go to 0.6 IMO. On most servers bufferpool ratio is <2% which is good enough.

Maybe monitor it for few days after change?